docs/FAQs.md

```diff
@@ -0,0 +1,42 @@
+# GitHub for Jira FAQs
+
+### Permissions
+
+**Q:** The permission scope for code and metadata suggests data is read to synchronize development information. Is code stored on Jira? Should I be concerned that, were the app to be compromised, an actor exfiltrate all our code from GitHub?
```

Comment:
Maybe make each question an `<h3>` to make them more prominent (and ditch the "Permissions" header, because we only seem to have one category anyways. That way we wouldn't need to use ugly `<br>`s in Markdown :)